apps/coscrad-frontend/src/utils/generic-components/presenters/multilingual-text-presenter.tsx

```diff
@@ -7,20 +18,46 @@ export interface MultilingualTextPresenterProps {
 export const MultilingualTextPresenter = ({
     text,
 }: MultilingualTextPresenterProps): JSX.Element => {
+    const { defaultLanguageCode } = useContext(ConfigurableContentContext);
+
     const { items } = text;
 
-    if (!Array.isArray(items)) {
+    const filteredItems = items.filter((item) => item.languageCode === defaultLanguageCode);
+
+    const translations = items.filter((items) => items.languageCode !== defaultLanguageCode);
+
+    if (!Array.isArray(filteredItems)) {
         throw new Error(`invalid input to Multlingual text!: ${text}`);
     }
 
     return (
         <>
-            {/* TODO We need to separate the original in some way */}
-            {items.map(({ languageCode, text, role }) => (
-                <div key={`${languageCode}-${role}`}>
-                    {`{${languageCode}} [${role}]`} {text}
-                </div>
-            ))}
+            <Box>
+                {filteredItems.map(({ languageCode, text, role }) => (
+                    <Box key={`${languageCode}-${role}`}>
+                        {text}
+                        <Typography variant="subtitle1" color={'secondary.dark'}>
+                            {`${languageCode}, '${role}'`}
+                        </Typography>
+                    </Box>
+                ))}
+            </Box>
+            <Box>
+                {translations.map(({ languageCode, text, role }) => (
```

Comment:
Shouldn't there be one top level accordion, whose items are generated with the map? I.e., shouldn't you iterate with translations.map inside the top level accordion container?